internal/graph/expand.go

```diff
@@ -59,7 +60,7 @@ func (ce *ConcurrentExpander) expandDirect(
 	return func(ctx context.Context, resultChan chan<- ExpandResult) {
 		requestRevision, err := decimal.NewFromString(req.Metadata.AtRevision)
 		if err != nil {
-			resultChan <- expandResultError(NewExpansionFailureErr(err), 0, 0)
```

Comment:
Why do the `(1 ,0)` and `(0, 0)` responses both use `emptyMetadata`?

Reply:
Because the `(1, 0)` was wrong after my change: it was adding `1` to the dispatch count, without a dispatch. Now the counter is handled at the entrypoint of the dispatch for each graph